connector/oidc/oidc.go

```diff
@@ -573,6 +584,11 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 		}
 	}
 
+	// add prefix/suffix to groups
+	for i, group := range groups {
+		groups[i] = c.groupsPrefix + group + c.groupsSuffix
+	}
```

Comment:
NIT
```suggestion
    if c.groupsPrefix != "" || c.groupsSuffix != "" {
		for i, group := range groups {
			groups[i] = c.groupsPrefix + group + c.groupsSuffix
		}
	}
```

Reply:
Added the nitpick, and ammended the commit